Networking.xcodeproj/project.pbxproj

```diff
@@ -548,12 +503,6 @@
 			mainGroup = E1FCED7D1E31358D00E3408F;
 			productRefGroup = E1FCED871E31358D00E3408F /* Products */;
 			projectDirPath = "";
-			projectReferences = (
```

Comment:
I rebased from `swift-4-support` to be able to merge, all these changes come from that

Reply:
Those changes are made in this branch, not related with base branch, whatever it is.

Reply:
Sorry, I thought those were new changes. I checked and the modifications in the `.pbxproject` come from reverting the commit that removed dependencies from the cartfile and then modified the project accordingly. This wasn't needed in the first place so what you are seeing is a lot of undoing of those changes.

Reply:
It seems the rebase I did wasn't done correctly or something because I'm seeing things in the cartfile that shouldn't be there. I'll commit a fix for all of this.